test/unittests/precompiles_expmod_test.cpp

```diff
@@ -22,14 +22,78 @@ TEST(expmod, test_vectors)
     const std::vector<TestCase> test_cases{
         {"", "", "", ""},
         {"", "", "00", "00"},
+        {"", "", "01", "00"},
```

Comment:
I added some test case for debugging. You can set them to evmone as a separate PR.